platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/cleanup/ProgramExecutionsCleaner.java

```diff
@@ -0,0 +1,37 @@
+package com.softicar.platform.core.module.program.execution.cleanup;
+
+import com.softicar.platform.common.date.DayTime;
+import com.softicar.platform.core.module.CoreI18n;
+import com.softicar.platform.core.module.event.SystemEventBuilder;
+import com.softicar.platform.core.module.event.severity.AGSystemEventSeverityEnum;
+import com.softicar.platform.core.module.program.Programs;
+import com.softicar.platform.core.module.program.execution.AGProgramExecution;
+import com.softicar.platform.core.module.program.state.AGProgramState;
+import com.softicar.platform.db.core.transaction.DbTransaction;
+
+public class ProgramExecutionsCleaner {
```

Comment:
there should be a `ProgramExecutionsCleanerTest`

Reply:
ohhh